ait/gui/__init__.py

```diff
@@ -750,6 +750,11 @@ def get_packet_delta(pkt_defn, packet):
         packet_states[pkt_defn.name]['dntoeu'] = {}
         dntoeus = {f.name: getattr(ait_pkt, f.name) for f in pkt_defn.fields if f.dntoeu is not None}
 
+        # get derivations
```

Comment:
Could we update the packet_states / delta directly while iterating over `pkt_defn.derivations` instead of creating the derivations dict first?